docs/contributing/development/continuous_integration.rst

```diff
@@ -315,60 +66,4 @@ In the near future we want to auto-update the citation guidelines in the
 Release pipeline
 ================
 
-Publishes a new release of TARDIS every sunday at 00:00 UTC. 
-
-
-Compare reference data pipeline
-===============================
-
-This pipeline compares two versions of the reference data. It's triggered manually via
-the Azure Pipelines web UI, or when a TARDIS contributor leaves the following comment
-on a pull request:
-::
-
-  /AzurePipelines run compare-refdata
-
-For brevity, you can comment using ``/azp`` instead of ``/AzurePipelines``.
-
-By default, generates new reference data for the ``HEAD`` of the pull request. Then, 
-compares against latest reference data stored in ``tardis-refdata`` repository. If
-you want to compare two different labels (SHAs, branches, tags, etc.) uncomment and
-set the ``ref1.hash`` and ``ref2.hash`` variables in 
-``.github/workflows/compare-refdata.yml`` on your pull request. For example:
-.. code-block:: yaml
-
-  ref1.hash: 'upstream/pr/11'
-  ref2.hash: 'upstream/master'
-
-The web UI also allows to compare any version of the reference data by providing those
-variables at runtime, but the access to the dashboard is restricted to a small group
-of developers.
-
-.. warning:: If using the Azure dashboard, do not define ``ref1.hash`` and ``ref2.hash``
-          between quotation marks or **the pipeline will fail**. This does not apply for
-          the YAML file.
-
-Finally, the report is uploaded to the
-`OpenSupernova.org server <http://opensupernova.org/~azuredevops/files/refdata-results/>`_
-following the ``<pr>/<commit>`` folder structure. If the pipeline fails, also a report is 
-generated, but not necessarily gives useful debug information (depends on which step the
-pipeline has failed).
-
-
-TARDIS Carsus Compatibility Check
-=================================
-The TARDIS Carsus Compatibility Check or the "Bridge" compares reference data 
-generated with different versions of Carsus. It consists of two jobs- a "carsus-build" job to 
-generate an atomic file with the latest version of Carsus and a "tardis-build" job 
-to generate a new reference data with it. These two reference data files are compared using the 
-`this notebook <https://github.com/tardis-sn/tardis-refdata/blob/master/notebooks/ref_data_compare_from_paths.ipynb>`_.
-The workflow has a ``workflow_dispatch`` event so that it can be triggered manually, but is also 
-triggered every week due to the "save-atomic-files" workflow. 
-
-
-The Save Atomic Files Workflow
-==============================
-The Save Atomic Files workflow runs every week but can also be triggered manually.
-It runs the "Bridge" and sends an artifact containing the generated atomic data file
-and the comparison notebook to Moria. This workflow has a separate job to indicate if the 
-bridge has failed.
+Publishes a new release of TARDIS every sunday at 00:00 UTC. 
```

Comment:
We should document the pre and post release workflows too but perhaps in a different pull request

Reply:
Yes.